Frontend/src/app/market/market.component.html

```diff
@@ -95,3 +85,55 @@
     </div>
 </div>
 
+<div class="container mt-4">
+  <div *ngIf="posts.length > 0; else noDataMessage" >
+  <div *ngFor="let post of posts; let i = index" class="card mb-4">
+    <div class="row no-gutters">
+      <div class="col-md-4">
+        <img [src]="post.picture" class="card-img" alt="Post Image">
+      </div>
+      <div class="col-md-8">
+        <div class="card-body">
+          <h5 class="card-title">{{ post.name }}</h5>
+          <p class="card-text"><strong>Year of Purchase:</strong> {{ post.year }}</p>
+          <p class="card-text"><strong>Contact Number:</strong> {{ post.contact }}</p>
+
+          <div class="row">
+            <div class="col-md-6">
+              <p class="card-text"><strong>Address:</strong> {{ post.address }}</p>
+            </div>
+            <div class="col-md-6">
+              <p class="card-text"><strong>Region:</strong> {{ post.region }}</p>
```

Comment:
use city instead of the region so the payload is match the backend variables

Reply:
Ok I will change that. Anything else.

Reply:
I am commiting once again